src/components/editor/PropertyTemplateOutline.jsx

```diff
@@ -11,6 +11,22 @@ import PropTypes from 'prop-types'
 import shortid from 'shortid'
 import ResourceProperty from "./ResourceProperty";
 
+export const AddResourceTemplate = (resourceTemplate, reduxPath) => {
```

Comment:
Should this be made lower-case since it's a regular function?

Reply:
Updated in latest commit.